PhysicsTools/PatAlgos/python/tools/jetTools.py

```diff
@@ -170,8 +170,8 @@ def setupSVClustering(btagInfo, svClustering, algo, rParam, fatJets=cms.InputTag
             btagInfo.groomedFatJets = groomedFatJets
 
 
-def setupBTagging(process, jetSource, pfCandidates, explicitJTA, pvSource, svSource, elSource, muSource, runIVF, loadStdRecoBTag, svClustering, fatJets, groomedFatJets,
-                  algo, rParam, btagDiscriminators, btagInfos, patJets, labelName, postfix):
+def setupBTagging(process, jetSource, pfCandidates, explicitJTA, pvSource, svSource, elSource, muSource, runIVF, hipMitigation, loadStdRecoBTag, svClustering, fatJets, groomedFatJets,
+                  algo, rParam, btagDiscriminators, btagInfos, patJets, labelName, btagPrefix, postfix):
```

Comment:
was the prefix really necessary?
I'm trying to understand if we could instead do away with just the available postfix functionality.
The number of changes triggered by the prefix need seems excessive.